lib/fluent/config/element.rb

```diff
@@ -213,6 +216,17 @@ def self.unescape_parameter(v)
         v.each_char { |c| result << LiteralParser.unescape_char(c) }
         result
       end
+
+      def set_target_worker_id(worker_id)
+        @target_worker_id = worker_id
+        @elements.each { |e|
+          e.set_target_worker_id(worker_id)
+        }
+      end
+
+      def has_target?
```

Comment:
`#has_target` doesn't show the correct meaning of this method. What's the target?
`#has_target_worker_id` looks better.